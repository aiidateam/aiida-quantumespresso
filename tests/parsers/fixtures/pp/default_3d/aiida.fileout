 Synthetic cubefile for testing the pp parser
Contains the selected quantity on a FFT grid
    8    0.000000    0.000000    0.000000
    8    0.337013    0.000000    0.000000
    8    0.000000    0.337013    0.000000
    8    0.000000    0.000000    0.337013
    6    6.000000    6.740256    6.740256    6.740256
    6    6.000000    6.740256    3.370128    3.370128
    6    6.000000    3.370128    6.740256    3.370128
    6    6.000000    3.370128    3.370128    6.740256
    6    6.000000    5.055192    5.055192    1.685064
    6    6.000000    5.055192    1.685064    5.055192
    6    6.000000    1.685064    5.055192    5.055192
    6    6.000000    1.685064    1.685064    1.685064
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
 -0.11488E+02 -0.92624E+01 -0.45999E+01 -0.11592E+01  0.24463E+00  0.77250E+00
  0.95788E+00  0.10154E+01
