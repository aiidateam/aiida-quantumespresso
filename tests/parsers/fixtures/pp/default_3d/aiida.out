
     Program POST-PROC v.6.4 starts on 22Oct2019 at 22:12:22

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org",
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =       4

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file C.pbe-n-kjpaw_psl.1.0.0.UPF: wavefunction(s)  2S 2P renormalized

     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         326     164     40                 8876     3132     386
     Max         327     165     41                 8881     3135     389
     Sum        1305     657    161                35513    12533    1551


     Check: negative core charge=   -0.000013

     Calling punch_plot, plot_num =  11
     Writing data to file  aiida.filplot
     Reading data from file  aiida.filplot

     Writing data to be plotted to file aiida.fileout
     Plot Type: 3D                     Output format: Gaussian cube

     POST-PROC    :      0.56s CPU      0.62s WALL


   This run was terminated on:  22:12:22  22Oct2019

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
